traitcuration/traits/datasources/clinvar.py

```diff
@@ -0,0 +1,77 @@
+import urllib.request
+import csv
+import gzip
+import shutil
+import itertools
+import os
+from ..models import Trait
+
+
+def extract_clinvar_data():
+    """
+    This function downloads the latest ClinVar TSV release data and extracts it into a 
+    'variant_summary.txt' file
+    """
+    print("Downloading ClinVar data...")
+    url = 'https://ftp.ncbi.nlm.nih.gov/pub/clinvar/tab_delimited/variant_summary.txt.gz'
+    urllib.request.urlretrieve(url, './variant_summary.txt.gz')
+    print("Extracting ClinVar data...")
+    with gzip.open('variant_summary.txt.gz', 'rb') as f_in:
+        with open('variant_summary.txt', 'wb') as f_out:
+            shutil.copyfileobj(f_in, f_out)
```

Comment:
I wonder if explicitly unpacking this file is necessary—presumably, inside `parse_trait_names_and_source_records` we could just call `gzip.open` directly on the gzipped file instead of `open`.

(However, downloading the file should be still left in a separate function.)

Reply:
You are right, I removed the extraction and added opening via gzip